.github/workflows/nightlies.yml

```diff
@@ -61,6 +63,9 @@ jobs:
           # We need to store credentials here
           persist-credentials: true
       - name: Commit and push
+        if: ${{ github.repository_owner == 'freedomofpress' }}
+        permissions:
```

Comment:
For this push we don't use `$GITHUB_TOKEN`, instead we use a pre-configured PAT that is stored as a secret (`${{ secrets.PUSH_TOKEN }}`), so I would assume that we don't need to even get write permissions, because we aren't using the workflow's own token.

IMO the easiest way to test it is to just merge it, if the workflow fails, then we know it's missing permissions and can revert or whatever. (Also please don't wait for me if this is in the path of any release stuff)